cloudquery-cloud/README.md

```diff
@@ -0,0 +1,38 @@
+# CloudQuery Cloud
+
+![datadog-integration][1]
+
+## Overview
+
+[CloudQuery][2] is an open-source, high-performance data integration framework built for developers, with support for a wide range of plugins.
```

Comment:
Within the overview section, we typically ask Partners to follow the below format: 
1. 1-2 sentences about what your platform does
2. 1-2 sentences about what this integration does and what kind of data it pulls/submits
3. 1-2 sentences about how this integration benefits customers

Could you be more specific about what CloudQuery does in your first sentence?